data/json/monsters/cyborgs.json

```
@@ -85,5 +85,94 @@
       "PATH_AVOID_DANGER_2",
       "PRIORITIZE_TARGETS"
     ]
+  },
+  {
+    "id": "mon_exodii_worker",
+    "type": "MONSTER",
+    "name": "Exodii worker",
+    "description": "This is a mostly humanoid robot equipped with various construction tools.",
+    "default_faction": "exodii",
+    "volume": "70000 ml",
+    "weight": 120000,
+    "species": [ "CYBORG" ],
+    "material": [ "steel" ],
+    "speed": 160,
+    "symbol": "e",
+    "morale": 20,
+	"agression": -30,
+    "hp": 125,
+    "color": "light_gray",
+    "melee_skill": 8,
+    "melee_dice": 2,
+    "melee_dice_sides": 6,
+    "melee_cut": 12,
+    "armor_bash": 12,
+    "armor_cut": 28,
+    "vision_day": 50,
+    "vision_night": 50,
+    "revert_to_itype": "e_scrap",
+    "anger_triggers": [ "FRIEND_ATTACKED", "FRIEND_DIED", "HURT" ],
+    "death_drops": { "subtype": "collection", "groups": [ [ "robots", 80 ] ] },
+    "death_function": [ "FIREBALL" ],
+    "harvest": "mon_broken_cyborg",
+    "flags": [ "SEES", "FLIES", "NOHEAD", "NO_BREATHE", "PATH_AVOID_DANGER_1", "HIT_AND_RUN" ]
+  },
+  {
+    "id": "mon_exodii_quad",
+    "type": "MONSTER",
+    "name": "Exodii quadruped",
+    "description": "This enormous quadrupedal robot seems to be cobbled together from parts, most of them unfamiliar to you.  It moves with a heavy, oddly graceful gait, its footsteps leaving shallow craters behind.  It bristles with an arsenal of weaponry, but doesn't seem in a particular rush to target you.",
+    "default_faction": "exodii",
+    "volume": "85000 ml",
+    "weight": 400000,
+    "species": [ "CYBORG" ],
+    "material": [ "steel" ],
+    "speed": 90,
+    "symbol": "M",
+    "aggression": -30,
+    "morale": 40,
+    "hp": 500,
+    "color": "light_gray",
+    "melee_skill": 10,
+    "melee_dice": 3,
+    "melee_dice_sides": 8,
+    "melee_cut": 12,
+    "armor_bash": 42,
+    "armor_cut": 48,
+    "vision_day": 50,
+    "vision_night": 50,
+    "revert_to_itype": "e_scrap",
+    "starting_ammo": { "545": 3000, "40x53mm_buckshot_m169": 30 },
```

```suggestion
    "starting_ammo": { "545": 3000, "40x53mm_buckshot_m169": 30 },
```
Are they supposed to be using Russian rounds? 🤔

---

I believe they are, they're not from this world :)

---

Really they should be using 30-08 rounds, or a similar caliber that had a chance but didn't become mainstream. This was a decent filler until I get someone to help me add exodii weapons and ammo.